[BUG] In the enemy type selection, aquatic entities are not selected by any type

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.1.5 git-02de191

### Operating System

Windows 10

### Minecraft Version

1.20.4

### Describe the bug

You cannot select aquatic entities as any type of enemy.

### Steps to reproduce

Enable all enemy types, and enable KillAura or ESP, etc., next to aquatic entities.

### Client Log

```shell
[00:47:11] [main/INFO]: Loading Minecraft 1.20.4 with Fabric Loader 0.15.7
[00:47:12] [ForkJoinPool-1-worker-1/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR1-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[00:47:12] [main/INFO]: Loading 124 mods:
	- bookshelf 23.0.5
	- customskinloader 14.19.1
	- disable_custom_worlds_advice 5.0
	- fabric-api 0.96.4+1.20.4
	   |-- fabric-api-base 0.4.36+78d798af4f
	   |-- fabric-api-lookup-api-v1 1.6.49+82b1bb3e4f
	   |-- fabric-biome-api-v1 13.0.16+78d798af4f
	   |-- fabric-block-api-v1 1.0.16+3e2216cb4f
	   |-- fabric-block-view-api-v2 1.0.4+78d798af4f
	   |-- fabric-blockrenderlayer-v1 1.1.46+78d798af4f
	   |-- fabric-client-tags-api-v1 1.1.7+78d798af4f
	   |-- fabric-command-api-v1 1.2.41+f71b366f4f
	   |-- fabric-command-api-v2 2.2.20+78d798af4f
	   |-- fabric-commands-v0 0.2.58+df3654b34f
	   |-- fabric-containers-v0 0.1.86+df3654b34f
	   |-- fabric-content-registries-v0 5.0.14+78d798af4f
	   |-- fabric-convention-tags-v1 1.5.10+78d798af4f
	   |-- fabric-crash-report-info-v1 0.2.23+78d798af4f
	   |-- fabric-data-attachment-api-v1 1.1.3+b90db5744f
	   |-- fabric-data-generation-api-v1 13.2.3+5c0133444f
	   |-- fabric-dimensions-v1 2.1.61+78d798af4f
	   |-- fabric-entity-events-v1 1.6.1+09fc25014f
	   |-- fabric-events-interaction-v0 0.7.1+389931eb4f
	   |-- fabric-events-lifecycle-v0 0.2.74+df3654b34f
	   |-- fabric-game-rule-api-v1 1.0.46+78d798af4f
	   |-- fabric-item-api-v1 2.2.0+d6f2b0844f
	   |-- fabric-item-group-api-v1 4.0.25+58f8c0124f
	   |-- fabric-key-binding-api-v1 1.0.41+78d798af4f
	   |-- fabric-keybindings-v0 0.2.39+df3654b34f
	   |-- fabric-lifecycle-events-v1 2.3.0+a67ffb5d4f
	   |-- fabric-loot-api-v2 2.1.8+78d798af4f
	   |-- fabric-message-api-v1 6.0.5+78d798af4f
	   |-- fabric-mining-level-api-v1 2.1.64+78d798af4f
	   |-- fabric-model-loading-api-v1 1.0.8+78d798af4f
	   |-- fabric-models-v0 0.4.7+9386d8a74f
	   |-- fabric-networking-api-v1 3.1.7+2e5ac5484f
	   |-- fabric-object-builder-api-v1 13.0.13+080016e44f
	   |-- fabric-particles-v1 1.1.7+78d798af4f
	   |-- fabric-recipe-api-v1 2.0.20+78d798af4f
	   |-- fabric-registry-sync-v0 4.0.19+58f8c0124f
	   |-- fabric-renderer-api-v1 3.2.4+78d798af4f
	   |-- fabric-renderer-indigo 1.5.4+78d798af4f
	   |-- fabric-renderer-registries-v1 3.2.53+df3654b34f
	   |-- fabric-rendering-data-attachment-v1 0.3.42+73761d2e4f
	   |-- fabric-rendering-fluids-v1 3.1.1+e761c6694f
	   |-- fabric-rendering-v0 1.1.56+df3654b34f
	   |-- fabric-rendering-v1 3.2.0+6fd945a04f
	   |-- fabric-resource-conditions-api-v1 2.3.14+78d798af4f
	   |-- fabric-resource-loader-v0 0.11.19+58f8c0124f
	   |-- fabric-screen-api-v1 2.0.17+78d798af4f
	   |-- fabric-screen-handler-api-v1 1.3.55+78d798af4f
	   |-- fabric-sound-api-v1 1.0.17+78d798af4f
	   |-- fabric-transfer-api-v1 4.0.11+e84342304f
	   \-- fabric-transitive-access-wideners-v1 5.0.14+78d798af4f
	- fabric-language-kotlin 1.10.18+kotlin.1.9.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.2
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.5.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
	- fabricloader 0.15.7
	   \-- mixinextras 0.3.5
	- ibeeditor 2.2.8
	- indium 1.0.30+mc1.20.4
	- java 17
	- krypton 0.2.6
	   \-- com_velocitypowered_velocity-native 3.2.0-SNAPSHOT
	- lambdabettergrass 1.5.2+1.20.1
	- lambdynlights 2.3.4+1.20.4
	   |-- pride 1.2.0+1.19.4
	   \-- spruceui 5.0.3+1.20.2
	- lazydfu 0.1.3
	- liquidbounce 0.1.5
	- lithium 0.12.1
	- logical_zoom 0.0.22
	- memoryleakfix 1.1.5
	- minecraft 1.20.4
	- modernfix 5.13.0+mc1.20.4
	- modmenu 9.0.0
	- nbt_ac 1.2
	- nbttooltips 0.1.4-1.20.x
	- nochatreports 1.20.4-v2.5.0
	- packetfixer 1.2.4
	- reeses-sodium-options 1.7.2+mc1.20.4-build.102
	- shulkerboxtooltip 4.0.8+1.20.4
	   \-- cloth-config 13.0.121
	        \-- cloth-basic-math 0.6.1
	- sodium 0.5.8+mc1.20.4
	- sodium-extra 0.5.4+mc1.20.4-build.116
	   |-- caffeineconfig 1.3.0+1.17
	   \-- crowdin-translate 1.4+1.19.3
	- starlight 1.1.3+fabric.f5dcd1a
	- toolstats 19.0.3
	- viafabricplus 3.0.6
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viabackwards-common 4.10.0-23w51b-SNAPSHOT
	   |-- com_viaversion_viaversion-common 4.10.0-23w51b-SNAPSHOT
	   |-- de_florianmichael_classic4j 2.0.1
	   |-- io_jsonwebtoken_jjwt-api 0.12.3
	   |-- io_jsonwebtoken_jjwt-gson 0.12.3
	   |-- io_jsonwebtoken_jjwt-impl 0.12.3
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_mcping 1.4.0
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.0
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.0
	   |-- net_lenni0451_reflect 1.3.1
	   |-- net_raphimc_minecraftauth 3.1.0
	   |-- net_raphimc_viaaprilfools 2.0.11-SNAPSHOT
	   |-- net_raphimc_viabedrock 0.0.4-SNAPSHOT
	   |-- net_raphimc_vialegacy 2.2.22-SNAPSHOT
	   |-- net_raphimc_vialoader 2.2.13-SNAPSHOT
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR1-SNAPSHOT
	   |-- org_iq80_leveldb_leveldb 0.12
	   |-- org_iq80_leveldb_leveldb-api 0.12
	   |-- org_lz4_lz4-pure-java 1.8.0
	   \-- org_yaml_snakeyaml 2.2
	- xaerominimap 23.9.7
[00:47:12] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/H:/Games/Play2/.minecraft/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[00:47:12] [main/INFO]: Compatibility level set to JAVA_17
[00:47:13] [main/INFO]: Loaded configuration file for Lithium: 129 options available, 0 override(s) found
[00:47:13] [main/WARN]: Reference map 'memoryleakfix-fabric-1.17+-fabric-refmap.json' for memoryleakfix-fabric.mixins.json could not be read. If this is a development environment you can ignore this message
[00:47:13] [main/INFO]: Loaded configuration file for ModernFix 5.13.0+mc1.20.4: 52 options available, 0 override(s) found
[00:47:13] [main/INFO]: Applying Nashorn fix
[00:47:13] [main/WARN]: Reference map 'PacketFixer-fabric-fabric-refmap.json' for packetfixer-fabric.mixins.json could not be read. If this is a development environment you can ignore this message
[00:47:13] [main/INFO]: Loaded configuration file for Sodium: 42 options available, 0 override(s) found
[00:47:13] [main/INFO]: Loaded configuration file for Sodium Extra: 34 options available, 0 override(s) found
[00:47:14] [main/WARN]: For can't fit X into 3 error fix (dev.tonimatas.packetfixer.mixins.Varint21FrameDecoderMixin). Delete Krypton or Pluto.
[00:47:14] [main/WARN]: For can't fit X into 3 error fix (dev.tonimatas.packetfixer.mixins.Varint21LengthFieldPrependerMixin). Delete Krypton or Pluto.
[00:47:14] [main/INFO]: [MemoryLeakFix] Will be applying 3 memory leak fixes!
[00:47:14] [main/INFO]: [MemoryLeakFix] Currently enabled memory leak fixes: [targetEntityLeak, biomeTemperatureLeak, hugeScreenshotLeak]
[00:47:14] [main/INFO]: Searching for graphics cards...
[00:47:14] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.5).
[00:47:14] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=UNKNOWN, name=RayLink Virtual Display Adapter, version=DriverVersion=1.0.2.0]
[00:47:14] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=NVIDIA, name=NVIDIA GeForce GTX 1050 Ti, version=DriverVersion=31.0.15.3758]
[00:47:14] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS]
[00:47:14] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[00:47:16] [main/INFO]: liquidbounce.mixins.json:minecraft.entity.MixinPlayerEntityAccessor from mod liquidbounce->@Accessor[FIELD_GETTER]::getTrackedMainArm()Lnet/minecraft/class_2940; should be static as its target is
[00:47:17] [main/INFO]: Injecting BlockStateBase cache population hook into getNeighborPathNodeType from me.jellysquid.mods.lithium.mixin.ai.pathing.AbstractBlockStateMixin
[00:47:17] [main/INFO]: Injecting BlockStateBase cache population hook into getPathNodeType from me.jellysquid.mods.lithium.mixin.ai.pathing.AbstractBlockStateMixin
[00:47:17] [main/INFO]: Injecting BlockStateBase cache population hook into getOpacityIfCached from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[00:47:17] [main/INFO]: Injecting BlockStateBase cache population hook into isConditionallyFullOpaque from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[00:47:17] [main/INFO]: Injecting BlockStateBase cache population hook into lithium$getAllFlags from me.jellysquid.mods.lithium.mixin.util.block_tracking.AbstractBlockStateMixin
[00:47:18] [main/WARN]: Method overwrite conflict for method_21740 in modernfix-common.mixins.json:perf.remove_biome_temperature_cache.BiomeMixin from mod modernfix, previously written by me.jellysquid.mods.lithium.mixin.world.temperature_cache.BiomeMixin. Skipping method.
[00:47:20] [main/INFO]: Vanilla bootstrap took 4909 milliseconds
[00:47:21] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[00:47:21] [Render thread/INFO]: Setting user: mumy_
[00:47:22] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[00:47:22] [Render thread/INFO]: Bypassed Mojang DFU
[00:47:22] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:pufferfish_bucket is now minecraft:bucket.
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:salmon_bucket is now minecraft:bucket.
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:cod_bucket is now minecraft:bucket.
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:tropical_fish_bucket is now minecraft:bucket.
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:axolotl_bucket is now minecraft:bucket.
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:powder_snow_bucket is now minecraft:bucket.
[00:47:22] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:tadpole_bucket is now minecraft:bucket.
[00:47:22] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[00:47:22] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[00:47:22] [Render thread/INFO]: Initializing IBE Editor - common
[00:47:22] [Render thread/INFO]: Setting up IBE Editor - common
[00:47:22] [Download-2/ERROR]: Failed to fetch user properties
com.mojang.authlib.exceptions.InvalidCredentialsException: Status: 401
	at com.mojang.authlib.exceptions.MinecraftClientHttpException.toAuthenticationException(MinecraftClientHttpException.java:59) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:150) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_310.method_55608(class_310.java:467) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.get(MinecraftClient.java:57) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:121) ~[authlib-6.0.52.jar:?]
	... 5 more
[00:47:22] [Render thread/INFO]: Common configuration loaded
[00:47:22] [Render thread/INFO]: Compression will use Java, encryption will use Java
[00:47:22] [Render thread/INFO]: NBTac v1.2 for Minecraft 1.20.4 [Fabric] - Author: mt1006 (mt1006x)
[00:47:22] [Render thread/INFO]: All 8 have been assigned properly
[00:47:22] [Render thread/INFO]: Packet Fixer has been initialized successfully
[00:47:23] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[00:47:23] [Render thread/INFO]: Initializing IBE Editor - client
[00:47:23] [Render thread/INFO]: Setting up IBE Editor - client
[00:47:23] [Render thread/INFO]: Client configuration loaded
[00:47:23] [Render thread/INFO]: Vault loaded
[00:47:23] [Render thread/INFO]: Krypton is now accelerating your Minecraft client's networking stack ?
[00:47:23] [Render thread/INFO]: Note that Krypton is most effective on servers, not the client.
[00:47:23] [Render thread/INFO]: [LambdaBetterGrass] Initializing LambdaBetterGrass...
[00:47:23] [Render thread/INFO]: [LambDynLights] Initializing LambDynamicLights...
[00:47:23] [Render thread/INFO]: [LambDynLights] Configuration loaded.
[00:47:23] [Worker-Main-1/INFO]: Checking mod updates...
[00:47:23] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:14464/
[00:47:23] [Render thread/INFO]: Saving config to H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.7\config\toolstats.json.
[00:47:23] [Render thread/INFO]: Loading Xaero's Minimap - Stage 1/2
[00:47:24] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: 1.7.2-1.7.5 (4)
[00:47:24] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 23w51b (766)
[00:47:24] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[00:47:24] [Render thread/INFO]: Backend library: LWJGL version 3.3.2-snapshot
[00:47:24] [Render thread/WARN]: Applying workaround: Prevent the NVIDIA OpenGL driver from using broken optimizations (NVIDIA_THREADED_OPTIMIZATIONS)
[00:47:24] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[00:47:24] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
[00:47:24] [Render thread/INFO]: OpenGL Version: 3.3.0 NVIDIA 537.58
[00:47:25] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[00:47:25] [Finalizer/WARN]: One or more BufferBuilders have been leaked, ModernFix will attempt to correct this.
[00:47:26] [Render thread/INFO]: Launching LiquidBounce v0.1.5 by CCBlueX
[00:47:26] [Render thread/INFO]: Loaded language en_us
[00:47:26] [Render thread/INFO]: Loaded language de_de
[00:47:26] [Render thread/INFO]: Loaded language ja_jp
[00:47:26] [Render thread/INFO]: Loaded language zh_cn
[00:47:26] [Render thread/INFO]: Loaded language ru_ru
[00:47:26] [Render thread/INFO]: Loaded language en_pt
[00:47:26] [Render thread/INFO]: Loaded language pt_br
[00:47:27] [Render thread/INFO]: Successfully extracted default theme
[00:47:27] [Render thread/INFO]: Inserted 5 components
[00:47:27] [Render thread/INFO]: Successfully loaded config 'enemies'.
[00:47:27] [Render thread/INFO]: (Chat) 
[00:47:27] [Render thread/INFO]: (Chat) ��f��lLiquid��9��lBounce ��8? ��7Auto ConfigAuto Config
[00:47:27] [Render thread/INFO]: Successfully loaded config 'modules'.
[00:47:27] [Render thread/INFO]: Successfully loaded config 'commands'.
[00:47:27] [Render thread/INFO]: Successfully loaded config 'friends'.
[00:47:27] [Render thread/ERROR]: Unable to deserialize value proxy
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:397) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1329) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1271) ~[gson-2.10.1.jar:?]
	at net.ccbluex.liquidbounce.config.Value.deserializeFrom(Value.kt:218) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:275) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:230) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:205) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:203) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:146) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:174) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dca000$liquidbounce$startClient(class_310.java:16847) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:165) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:89) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:386) ~[gson-2.10.1.jar:?]
	... 24 more
[00:47:27] [Render thread/INFO]: Successfully loaded config 'proxy'.
[00:47:27] [Render thread/INFO]: Successfully loaded config 'accounts'.
[00:47:27] [Render thread/INFO]: Successfully loaded config 'tabs'.
[00:47:27] [Render thread/INFO]: Successfully loaded config 'lang'.
[00:47:27] [Render thread/INFO]: Successfully loaded config 'storage'.
[00:47:28] [netty-websocket/INFO]: Starting Netty server...
[00:47:28] [nioEventLoopGroup-3-1/INFO]: [id: 0x6e010127] REGISTERED
[00:47:28] [nioEventLoopGroup-3-1/INFO]: [id: 0x6e010127] BIND: 0.0.0.0/0.0.0.0:15000
[00:47:28] [nioEventLoopGroup-3-1/INFO]: [id: 0x6e010127, L:/[0:0:0:0:0:0:0:0]:15000] ACTIVE
[00:47:28] [netty-websocket/INFO]: Netty server started on port 15000.
[00:47:28] [Render thread/INFO]: java-cef commit: e91a133673f9524c5f7db9bdbad1aff61b3887d7
[00:47:28] [Render thread/INFO]: Task: Downloading Checksum with progress 0.0 %
[00:47:28] [Render thread/INFO]: https://dl.liquidbounce.net/resources/java-cef-builds/e91a133673f9524c5f7db9bdbad1aff61b3887d7/windows_amd64.tar.gz.sha256 -> H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.7\LiquidBounce\mcef\libraries\windows_amd64.tar.gz.sha256.temp
[00:47:30] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor!
[00:47:31] [Render thread/INFO]: Checksum matches: true
[00:47:31] [Render thread/INFO]: Platform directory exists: true
[00:47:31] [Render thread/INFO]: Initializing CEF on windows_amd64...
[00:47:31] [Render thread/INFO]: [STDOUT]: initialize on Thread[Render thread,5,main] with library path H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.7\LiquidBounce\mcef\libraries\windows_amd64
[00:47:31] [Render thread/INFO]: Chromium Embedded Framework initialized
[00:47:31] [Render thread/INFO]: Browser is ready.
[00:47:31] [Render thread/INFO]: Successfully loaded client!
[00:47:31] [Render thread/INFO]: Invalidating pack caches
[00:47:31] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, bookshelf, caffeineconfig, cloth-config, com_viaversion_viabackwards-common, com_viaversion_viaversion-common, disable_custom_worlds_advice, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, ibeeditor, indium, krypton, lambdabettergrass, lambdynlights, lazydfu, liquidbounce, lithium, logical_zoom, modernfix, modmenu, nbt_ac, nbttooltips, net_raphimc_viaaprilfools, net_raphimc_viabedrock, net_raphimc_vialegacy, nochatreports, packetfixer, pride, reeses-sodium-options, shulkerboxtooltip, sodium, sodium-extra, spruceui, starlight, toolstats, viafabricplus, xaerominimap
[00:47:31] [Render thread/INFO]: [LambdaBetterGrass] Inject generated resource pack.
[00:47:31] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[00:47:31] [Worker-ResourceReload-1/INFO]: Found unifont_all_no_pua-15.1.04.hex, loading
[00:47:31] [Worker-Main-2/ERROR]: Failed to retrieve profile key pair
com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.post(MinecraftClient.java:65) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.handler$cfl000$viafabricplus$storeLegacyPublicKeySignature(YggdrasilUserApiService.java:1050) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.getKeyPair(YggdrasilUserApiService.java) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_7434.method_43605(class_7434.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_7434.method_44291(class_7434.java:79) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[00:47:31] [Worker-Main-2/ERROR]: Failed to retrieve profile key pair
com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.post(MinecraftClient.java:65) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.handler$cfl000$viafabricplus$storeLegacyPublicKeySignature(YggdrasilUserApiService.java:1050) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.getKeyPair(YggdrasilUserApiService.java) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_7434.method_43605(class_7434.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_7434.method_44291(class_7434.java:79) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[00:47:31] [Render thread/INFO]: Created item group Heads
[00:47:31] [Render thread/INFO]: Created item group Exploits
[00:47:31] [Render thread/INFO]: Created item group Containers
[00:47:31] [Render thread/INFO]: Successfully setup tabs
[00:47:31] [Render thread/INFO]: Loading Xaero's Minimap - Stage 2/2
[00:47:35] [Render thread/ERROR]: suppressed exception
java.net.SocketTimeoutException: Read timed out
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2070) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2065) ~[?:?]
	at java.security.AccessController.doPrivileged(AccessController.java:569) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:2064) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1612) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1592) ~[?:?]
	at xaero.common.misc.Internet.checkModVersion(Internet.java:55) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	at xaero.minimap.XaeroMinimap.tryLoadLater(XaeroMinimap.java:272) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	at net.minecraft.class_310.handler$daj002$xaerominimap$onTickStart(class_310.java:15768) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1289) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.net.SocketTimeoutException: Read timed out
	at sun.nio.ch.NioSocketImpl.timedRead(NioSocketImpl.java:288) ~[?:?]
	at sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:314) ~[?:?]
	at sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:355) ~[?:?]
	at sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:808) ~[?:?]
	at java.net.Socket$SocketInputStream.read(Socket.java:976) ~[?:?]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:244) ~[?:?]
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:284) ~[?:?]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:343) ~[?:?]
	at sun.net.www.http.HttpClient.parseHTTPHeader(HttpClient.java:826) ~[?:?]
	at sun.net.www.http.HttpClient.parseHTTP(HttpClient.java:761) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1691) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1592) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getHeaderField(HttpURLConnection.java:3269) ~[?:?]
	at java.net.URLConnection.getHeaderFieldLong(URLConnection.java:639) ~[?:?]
	at java.net.URLConnection.getContentLengthLong(URLConnection.java:509) ~[?:?]
	at xaero.common.misc.Internet.checkModVersion(Internet.java:53) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	... 15 more
[00:47:35] [Render thread/INFO]: Registered player tracker system: minimap_synced
[00:47:35] [Render thread/INFO]: No Optifine!
[00:47:35] [Render thread/INFO]: Xaero's Minimap: No Vivecraft!
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushAttrib is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glLoadIdentity is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glOrtho is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glLoadIdentity is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPopMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPopMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPopAttrib is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushAttrib is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glLoadIdentity is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glOrtho is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glLoadIdentity is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPopMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPopMatrix is deprecated and not available in preview contexts.'
[00:47:35] [Thread-17/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPopAttrib is deprecated and not available in preview contexts.'
[00:47:35] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[00:47:35] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[00:47:35] [Render thread/WARN]: Missing sound for event: minecraft:entity.generic.wind_burst
[00:47:35] [nioEventLoopGroup-3-1/INFO]: [id: 0x6e010127, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xa4971ebd, L:/127.0.0.1:15000 - R:/127.0.0.1:14482]
[00:47:35] [nioEventLoopGroup-3-1/INFO]: [id: 0x6e010127, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[00:47:35] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on ������ (Realtek High Definition Audio)
[00:47:35] [Render thread/INFO]: Sound engine started
[00:47:35] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[00:47:35] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[00:47:35] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[00:47:35] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[00:47:35] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[00:47:35] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[00:47:35] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[00:47:35] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[00:47:35] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[00:47:36] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[00:47:36] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[00:47:36] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[00:47:36] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[00:47:36] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[00:47:36] [Render thread/INFO]: Loading fonts...
[00:47:36] [Render thread/INFO]: Loading queued font Montserrat
[00:47:36] [Render thread/INFO]: Loaded fonts successfully!
[00:47:40] [Render thread/INFO]: Update available! Please download the latest version from https://liquidbounce.net/
[00:47:40] [Render thread/INFO]: Loading Discord IPC configuration...
[00:47:43] [Render thread/INFO]: Loaded Discord IPC configuration.
[00:47:43] [Render thread/INFO]: Refreshing local IP info...
[00:47:44] [Render thread/INFO]: Loading heads...
[00:47:44] [nioEventLoopGroup-5-1/INFO]: IP Info [CN, AS17816 China Unicom IP network China169 Guangdong province]
[00:47:50] [nioEventLoopGroup-4-1/WARN]: An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
java.net.SocketException: Connection reset
	at sun.nio.ch.SocketChannelImpl.throwConnectionReset(SocketChannelImpl.java:394) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:426) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:254) ~[netty-buffer-4.1.97.Final.jar:?]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.97.Final.jar:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.97.Final.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.97.Final.jar:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.97.Final.jar:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
[00:47:51] [Render thread/INFO]: Successfully loaded 956 heads from the database
[00:47:51] [Render thread/INFO]: Loading settings list from API...
[00:47:57] [Render thread/INFO]: Loaded 25 settings from API.
[00:47:57] [Render thread/INFO]: Successfully loaded 233 cape carriers.
[00:47:57] [Thread-15/INFO]: Finished in: 35 ms [26146 ms with interruption]
[00:47:57] [Render thread/INFO]: (Chat) ��f��lLiquid��9��lBounce ��8? ��7UI desync detected, it should now be fixed!UI desync detected, it should now be fixed!
[00:47:57] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@16a76bfe, url=http://127.0.0.1:15000/default/#/)
...
```


### Screenshots

![2024-03-02_00 57 20](https://github.com/CCBlueX/LiquidBounce/assets/161578211/dfa110da-0d68-4b31-a32b-e823b99c66b4)
![2024-03-02_00 57 33](https://github.com/CCBlueX/LiquidBounce/assets/161578211/8f88ae20-836a-4e17-9a34-681c48591e5d)

--- a/src/main/kotlin/net/ccbluex/liquidbounce/utils/combat/CombatExtensions.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/utils/combat/CombatExtensions.kt
@@ -29,7 +29,6 @@ import net.ccbluex.liquidbounce.features.module.modules.misc.ModuleTeams
 import net.ccbluex.liquidbounce.features.module.modules.misc.antibot.ModuleAntiBot
 import net.ccbluex.liquidbounce.features.module.modules.render.murdermystery.ModuleMurderMystery
 import net.ccbluex.liquidbounce.utils.client.*
-import net.ccbluex.liquidbounce.utils.client.player
 import net.ccbluex.liquidbounce.utils.entity.squaredBoxedDistanceTo
 import net.ccbluex.liquidbounce.utils.kotlin.toDouble
 import net.minecraft.client.network.AbstractClientPlayerEntity
@@ -39,8 +38,10 @@ import net.minecraft.entity.Entity
 import net.minecraft.entity.EntityGroup
 import net.minecraft.entity.LivingEntity
 import net.minecraft.entity.attribute.EntityAttributes
+import net.minecraft.entity.mob.Angerable
 import net.minecraft.entity.mob.HostileEntity
 import net.minecraft.entity.mob.Monster
+import net.minecraft.entity.mob.WaterCreatureEntity
 import net.minecraft.entity.passive.PassiveEntity
 import net.minecraft.entity.player.PlayerEntity
 import net.minecraft.network.packet.c2s.play.PlayerInteractEntityC2SPacket
@@ -71,6 +72,12 @@ class EnemyConfigurable : Configurable("Enemies") {
     // Hostile mobs (like skeletons and zombies) should be considered as an enemy
     var hostile by boolean("Hostile", true)
 
+    // Angerable mobs (like wolfs) should be considered as an enemy
+    val angerable by boolean("Angerable", true)
+
+    // Water Creature mobs should be considered as an enemy
+    val waterCreature by boolean("WaterCreature", true)
+
     // Passive mobs (like cows, pigs and so on) should be considered as an enemy
     var passive by boolean("Passive", false)
 
@@ -96,19 +103,19 @@ class EnemyConfigurable : Configurable("Enemies") {
     fun isTargeted(suspect: Entity, attackable: Boolean = false): Boolean {
         // Check if the enemy is living and not dead (or ignore being dead)
         if (suspect is LivingEntity && (dead || suspect.isAlive)) {
-            // Check if enemy is sleeping (or ignore being sleeping)
-            if (suspect.isSleeping && !sleeping) {
-                return false
-            }
-
             // Check if enemy is invisible (or ignore being invisible)
             if (invisible || !suspect.isInvisible) {
-                if (attackable && ModuleTeams.isInClientPlayersTeam(suspect)) {
-                    return false
-                }
-
                 // Check if enemy is a player and should be considered as an enemy
                 if (suspect is PlayerEntity && suspect != mc.player) {
+                    if (attackable && ModuleTeams.isInClientPlayersTeam(suspect)) {
+                        return false
+                    }
+
+                    // Check if enemy is sleeping (or ignore being sleeping)
+                    if (suspect.isSleeping && !sleeping) {
+                        return false
+                    }
+
                     if (attackable && !friends && FriendManager.isFriend(suspect)) {
                         return false
                     }
@@ -130,10 +137,14 @@ class EnemyConfigurable : Configurable("Enemies") {
                     }
 
                     return players
+                } else if (suspect is WaterCreatureEntity) {
+                    return waterCreature
                 } else if (suspect is PassiveEntity) {
                     return passive
                 } else if (suspect is HostileEntity || suspect is Monster) {
                     return hostile
+                } else if (suspect is Angerable) {
+                    return angerable
                 }
             }
         }

